protocol-units/bridge/chains/movement/src/lib.rs

```diff
@@ -308,33 +303,35 @@ impl MovementClient {
 		// Update the content of Move.toml with the new addresses
 		let updated_content = move_toml_content
 			.lines()
-			.map(|line| match line {
-				_ if line.starts_with("resource_addr = ") => {
-					format!(r#"resource_addr = "{}""#, formatted_resource_address)
-				}
-				_ if line.starts_with("atomic_bridge = ") => {
-					format!(r#"atomic_bridge = "{}""#, formatted_resource_address)
-				}
-				_ if line.starts_with("moveth = ") => {
-					format!(r#"moveth = "{}""#, formatted_resource_address)
-				}
-				_ if line.starts_with("master_minter = ") => {
-					format!(r#"master_minter = "{}""#, formatted_resource_address)
-				}
-				_ if line.starts_with("minter = ") => {
-					format!(r#"minter = "{}""#, formatted_resource_address)
-				}
-				_ if line.starts_with("admin = ") => {
-					format!(r#"admin = "{}""#, formatted_resource_address)
-				}
-				_ if line.starts_with("origin_addr = ") => {
-					format!(r#"origin_addr = "{}""#, address)
-				}
-				_ if line.starts_with("source_account = ") => {
-					format!(r#"source_account = "{}""#, address)
-				}
-				_ => line.to_string(),
-			})
+			.map(|line| {
```

Comment:
As this is testing code, can we move this lofgic out of the main `lib.rs` into utils and feature flag the calling method, I think its `publish_for_test` as #[cfg(test)]`, then this code will only get build when running `cargo test.